fvm/context.go

```diff
@@ -47,6 +49,7 @@ type Context struct {
 	Logger                        zerolog.Logger
 	ReusableCadenceRuntimePool    reusableRuntime.ReusableCadenceRuntimePool
 	BlockPrograms                 *programs.Programs
+	MeterParameters               *meter.MeterParameters
```

Comment:
Meter parameters should always be non-nil. By default they should be equal to whatever `meter.DefaultParameters()` uses.

It might also be better if they were not a reference.